backend/stet/domain/strings.py

```diff
@@ -0,0 +1,23 @@
+TRANSLATED_HEADER_PHRASES_TABLE: dict[str, str] = {
```

Comment:
This approach is fine for now, but I'd love to explore the idea of encoding this information in the template doc rather than hard-coding it in the source.  In that scenario, when there are corrections/updates to the translated text we can just update the DOCX instead of having to recompile.

Reply:
Good idea, I will work toward that, thank you.